api/v1alpha1/instaslice_types.go

```diff
@@ -21,15 +21,29 @@ import (
 )
 
 type Mig struct {
-	Placements     []Placement `json:"placements,omitempty"`
```

Comment:
Why go from omitempty to required?

Reply:
placements is needed for packing, we dont want any CR with empty placements